test/card/basic/card.spec.js

```diff
@@ -85,6 +85,19 @@ describe('Component: pfCard', function() {
 
     });
 
+    it("should show and hide the spinner", function() {
+
+      // by default, spinner should be hidden
+      element = compileCard('<pf-card head-title="My card title" sub-title="My card subtitle title" show-top-border="false">Inner content goes here</pf-card>', $scope);
+      cardClass = angular.element(element).find('.spinner-lg');
+      expect(cardClass.length).toBe(0);
+
+      // setting to true should show spinner
+      element = compileCard('<pf-card head-title="My card title" show-spinner="true" sub-title="My card subtitle title" show-top-border="false">Inner content goes here</pf-card>', $scope);
+      cardClass = angular.element(element).find('.spinner-lg');
+      expect(cardClass.length).toBe(1);
+    });
+
```

Comment:
Might be better to test the binding. Use show-spinner="dataLoading" then change the value and test again.

Reply:
Sounds good. I'll go back to this